src/content/docs/opentelemetry/best-practices/opentelemetry-manage-ingest-volume.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: Manage OpenTelemetry Ingest Volume
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+  - OTLP
+metaDescription: Manage OpenTelemetry Ingest Volume
+freshnessValidatedDate: never
+redirects:
+---
+
+One of OpenTelemetry's core strengths is its rich set of tools providing unparalleled control over your telemetry data pipeline. This control complements New Relic's [consumption-based pricing](https://newrelic.com/pricing#pricing_plan-data) model.
+
+This page talks about a variety of concepts that contribute to data volume when using OpenTelemetry with New Relic, and tools / patterns available to manage your telemetry data pipeline. It's organized into sections talking about key concepts contributing to data volume for [resource](#concept-resources), [traces](#concept-traces), [metrics](#concept-metrics), and [logs](#concept-logs), followed by a [catalog of tools](#tool-catalog) available for helping manage volume.
+
+## Key concepts: Resources [#concept-resources]
+
+[OTLP](https://opentelemetry.io/docs/specs/otlp/) (OpenTeLemetry Protocol) defines a similar hierarchical message structure across all signals, which avoid repetition at the protocol level by sharing information across records.
+
+* Each export request contains many `Resource{SignalRecord}`s
+* Each `Resource{SignalRecord}` contains many `Scope{SignalRecord}`s
+* Each `Scope{SignalRecord}` contains many `{SignalRecord}`s
+
+Where `{SignalRecord}` is `Span`, `Metric`, and `LogRecord`.
+
+As discussed in the pages discussing ingestion of [Traces](/docs/opentelemetry/best-practices/opentelemetry-best-practices-traces/), [Metrics](/docs/opentelemetry/best-practices/opentelemetry-best-practices-metrics/), and [Logs](/docs/opentelemetry/best-practices/opentelemetry-best-practices-logs/) sent to the [New Relic OTLP Endpoint](/docs/opentelemetry/best-practices/opentelemetry-otlp/), this hierarchical structure is flattened during New Relic and each resource attribute is denormalized onto individual `Span`, `Metric`, and `Log` records.
```

Comment:
> Instead of relying on doc links here, summarize what those pages are saying about ingestion and only link out if necessary (in a "for more information on X, see this specific section" structure) 

Fair enough. I like this phrasing, but do think the links are necessary. Will add them in a "for more info on X" structure.

> The phrase "flattened during New Relic" is a little confusing. Do you mean "flattened when exported to New Relic"?

I meant "flattened during ingestion in New Relic", but I like your phrasing better. 

> What does "denormalized" mean in this context?

Denormalized is a computer science term that means that the same piece of data is copied to many places. In a normalized representation, each piece of data exists in one place without repetition. NoSQL datastore solutions (like NRDB and many others) often rely on denormalization to avoid having to "join" data together. In new relic, instead of having a record for each resource, where each resource has many related metrics, and each metric has many related points, we have a rows for each point. And each point row has the information about its parent metric and resource denomarlized onto it so that all the information about the point is self contained.